### DISCUSSION TITLE
What are the Best Resources to Learn about Git and Github?

### DISCUSSION THREAD

**Discussion by Mafia-1911** [community/community#142726] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Give your suggestions !

**Comment by Mafia-1911** [comment]
I believe the github docs to the best one by far , But still a lot of Youtube Videos are there too !

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/142727#discussioncomment-11078546) for reference to why this discussion was closed and locked. Thanks!
